src/components/vm/consoles/consoles.jsx

```diff
@@ -133,27 +153,46 @@ export const ConsoleCard = ({ state, vm, config, onAddErrorNotification, isExpan
                         inactiveConsoleDetail={inactive_vnc}
                         spiceDetail={spice}
                         onAddErrorNotification={onAddErrorNotification}
-                        isExpanded={isExpanded} />
+                        isExpanded={isExpanded || isStandalone}
+                        onRemoteSizeChanged={(w, h, mode) => {
+                            if (lastVncRemoteSize.current[0] == w &&
+                                lastVncRemoteSize.current[1] == h)
+                                return;
+                            lastVncRemoteSize.current = [w, h];
+                            if (isStandalone && mode == "none") {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2254-7fcc4fae-20250807-092345-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)